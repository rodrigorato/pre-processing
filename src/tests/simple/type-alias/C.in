alias S 		= String;
public class Foo {
	public static S foo(S a, Integer b) {		
	}
}