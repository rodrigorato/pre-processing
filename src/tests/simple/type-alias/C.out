
public class Foo {
	public static String foo(String a, Integer b) {		
	}
}